### PR TITLE
Cars-assemble: fix tests failing due to floating point precision

### PR DESCRIPTION
**Pull request by tasxatzial**
A simple fix for the issue described below:

https://forum.exercism.org/t/cant-solve-cars-assemble-because-of-floating-point-precision/9242

The current tests check if two double numbers x, y are equal, and as a result some of them are failing when the numbers aren't identical. Two possible solutions: 
* Check if the two numbers are close enough using a check like `abs(x-y)<epsilon`.
* Truncate doubles to floats. This lowers the required precision required for the tests to pass.

This PR employs the second approach.

### CODE REVIEW

**Comment by tasxatzial** [comment]
I've just realized that my PR introduces a small typo . I'll correct it when it's reopened.

File: exercises/concept/cars-assemble/test/cars_assemble_test.clj [hunk 1]
```diff
@@ -2,29 +2,33 @@
...
-    (is (= 221.0 (cars-assemble/production-rate 1)))))
+    (is (float-equal? 221.0 (cars-assemble/production-rate 1)))))
 
 (deftest ^{:task 1} production-rate-speed-4-test
   (testing "Production rate for speed 4"
-    (is (= 884.0 (cars-assemble/production-rate 4)))))
+    (is (float-equal? 884.0 (cars-assemble/production-rate 4)))))
 
-(deftest ^{:task 1} production-rate-speed-7-test
+(deftest ^{:task 1} production-rate-speeed-7-test
```
Comment (IsaacG) [inline exercises/concept/cars-assemble/test/cars_assemble_test.clj:21]: Is this change intentional? The validate exercise step should fail here.

Comment (tasxatzial) [reply exercises/concept/cars-assemble/test/cars_assemble_test.clj:21]: That's the typo i'm talking about.

**Comment by tasxatzial** [comment]
@bobbicodes Do you have some time to review this one?

**Review by bobbicodes — APPROVED** [review]
Yeah I suppose this would fix a lot of problems with this exercise, which is not supposed to be a challenge, it *should* be simple at this point. I'll wait to merge it until the typo is fixed.

**Comment by tasxatzial** [comment]
It's ready. Thanks!
